### ISSUE TITLE
Shortcut key to enable the capture mode

### ISSUE THREAD

**Issue by brunoicq** [dynobo/normcap#659]
### Describe your problem:

Every time I need to capture a portion of the screen, I have to open the application or, to make it easier, keep it in the system tray. However, even this way, I still have to open the system tray and select the Normcap icon to start the capture.

### Solution you'd like to see:

Inside the settings menu, there should be an option where users can set a shortcut key to start the capture process. This will make the application much more productive.

**Comment by dangkhoa1612** [comment]
I completely agree. All functions are great, just need to add more shortcut key to be perfect

**Comment by GangstersPairOfDice** [comment]
Agree, although it is open source, maybe create a pull request?

Anyway a current hack you can do on Mac OS X is to use Automator ( comes preinstalled with Mac OS X )

There's a guide you can follow [here](https://www.idownloadblog.com/2021/01/15/launch-app-keyboard-shortcut-mac/)

**Comment by kurtmckee** [comment]
It appears that this has been considered and rejected by the author.

https://github.com/dynobo/normcap/blob/main/adr/004-do-not-implement-hotkey.md#decision-outcome

**Comment by tiktak888** [comment]
no problem , just use custom key shortcuts to run the program
